### PR TITLE
Fixes Accept header cannot be overridden issue

### PR DESCRIPTION
**Pull request by spiritinlife**
Used setValue instead of addValue so that user can override headers.

### CODE REVIEW

File: ios/Classes/SwiftFlutterUploaderPlugin.swift [hunk 1]
```diff
@@ -278,11 +278,12 @@ public class SwiftFlutterUploaderPlugin: NSObject, FlutterPlugin, URLSessionTask
                                                        completion completionHandler:@escaping (URLSessionUploadTask?, FlutterError?) -> Void) {
         let request = NSMutableURLRequest(url: url)
         request.httpMethod = method
-        request.addValue("*/*", forHTTPHeaderField: "Accept")
+        request.setValue("application/json; charset=utf-8", forHTTPHeaderField: "Accept")
```
Comment (ened) [inline ios/Classes/SwiftFlutterUploaderPlugin.swift:281]: The content type is configurable below but fixed to `application/JSON` here, can this be changed so it’s configurable both times?

File: ios/Classes/SwiftFlutterUploaderPlugin.swift [hunk 2]
```diff
@@ -278,11 +278,12 @@ public class SwiftFlutterUploaderPlugin: NSObject, FlutterPlugin, URLSessionTask
...
 
         headers?.forEach { (key, value) in
+            print(value)
```
Comment (ened) [inline ios/Classes/SwiftFlutterUploaderPlugin.swift:284]: Please remove

File: ios/Classes/SwiftFlutterUploaderPlugin.swift (hunk 1 above)
Comment (spiritinlife) [reply ios/Classes/SwiftFlutterUploaderPlugin.swift:281]: So you mean to set it to `request.setValue("*/*", forHTTPHeaderField: "Accept")` right ?

Comment (ened) [reply ios/Classes/SwiftFlutterUploaderPlugin.swift:281]: Yes.. in theory the plugin should not expect/restrict a server response.

Comment (spiritinlife) [reply ios/Classes/SwiftFlutterUploaderPlugin.swift:281]: Yup my bad. Will change it soon.

**Pull request merged**